internal/testutil/acc/config_formatter_test.go

```diff
@@ -104,3 +108,257 @@ func TestFormatToHCLLifecycleIgnore(t *testing.T) {
 		})
 	}
 }
+
+var standardClusterResource = `
+resource "mongodbatlas_advanced_cluster" "cluster_info" {
```

Comment:
i find cluster_info confusing as it's a resource not a data source.   We normally call it test so the resource is mongodbatlas_advanced_cluster.test, but you can use a different name

Reply:
I named it after the function. Seemed clearer to separate from an otherwise created cluster. But no strong opinion